### PR TITLE
Removes Boxer Strain

### PR DESCRIPTION
**Pull request by Chainsawicus**
## About The Pull Request

This PR seeks to remove the Boxer strain, including its unique abilities Jab and Uppercut, since they are unused without the strain.

Let me know if I missed anything.

## Why It's Good For The Game

Boxer is, in terms of actual usability, very weak. Its core design revolves around specifically taking one-on-one fights, and it uses a T2 slot to do this, which ends up making it somewhat of a meme strain, because all combat castes are designed to & expected to be able to 1v1 a marine with relative ease. 

Other problems with Boxer include its abusability, the use of third party scripts turn the strain from a weak but admittedly fun strain into an absolute untouchable monster that deletes people extremely quickly. Unfortunately, this problem is not an easy fix, some would assume that removing the reset function and just making its cooldowns incredibly short would alleviate this issue, but all this would do is further drive the strain's playability into the ground, while also removing the central point and theme of the strain. So, Boxer is left with two options: a rework, or a removal. I have opted for the latter.

## Changelog

:cl: ChainsawMullet
del: Warrior's Boxer strain has been removed.
/:cl:

### CODE REVIEW

**Comment by stanalbatross** [comment]
caste that has no purpose other than abusing its abilities on preds and cheating

good riddance

**Comment by c4xmaniac** [comment]
Boxer is cool but I agree, everyone remembers the infamous XX boxer who would kill 1/4 of the marine pop every round, and every pred/synth knows the amazing feeling of seeing a warrior come on screen and go boxer instantly upon seeing you.

**Comment by Tisx1** [comment]
**UNFATHOMABLY BASED**

**Comment by Diegoflores31** [comment]
lmao

**Comment by Zangeiti** [comment]
SPEEDMERGE... NOW!!!

**Comment by Carnage068** [comment]
ding ding ding...

**Comment by Carnage068** [comment]
If it was only viable for people who were legit cheating, I guess it's fair that it's being removed, but I will really fucking miss it. It was one of the few things that always made me laugh.

File: code/modules/mob/living/carbon/xenomorph/abilities/warrior/warrior_powers.dm [hunk 1]
```diff
@@ -134,12 +134,6 @@
 
 	var/mob/living/carbon/H = A
 
-	if (distance > 1 && X.mutation_type == WARRIOR_BOXER)
-		step_towards(X, H, 1)
-
-	if (!X.Adjacent(H))
-		return
```
Comment (Shad0vvs) [inline code/modules/mob/living/carbon/xenomorph/abilities/warrior/warrior_powers.dm:141]: Wild man.

**Pull request merged**